src/GraphQL/Utilities/AstPrinter.cs

```diff
@@ -264,6 +264,12 @@ public AstPrintVisitor()
                 c.Field(x => x.Value);
                 c.Print(f => f.Arg(x => x.Value));
             });
+
+            Config<NullValue>(c =>
+            {
+                c.Print(f => null);
```

Comment:
Are you sure?  If you do that you end up with null in a string which correctly fails a couple of unit tests:

Message: Assert.Equal() Failure
                                 ↓ (pos 29307)
Expected: ···    "defaultValue": null\r\n          }\r\n        ]\r\n      },\r\n ···
Actual:   ···    "defaultValue": "null"\r\n          }\r\n        ]\r\n      },\r···
                                 ↑ (pos 29307)